doc/rules/function_notation/multiline_promoted_properties.rst

```diff
@@ -0,0 +1,77 @@
+======================================
+Rule ``multiline_promoted_properties``
+======================================
+
+Promoted properties must be on separate lines.
+
+Configuration
+-------------
+
+``keep_blank_lines``
+~~~~~~~~~~~~~~~~~~~~
+
+Whether to keep blank lines between properties.
+
+Allowed types: ``bool``
```

Comment:
curious - what is the use case for this config?

Reply:
https://github.com/kubawerlos/php-cs-fixer-custom-fixers/issues/873